src/pages/cli/storage/override.mdx

```diff
@@ -14,46 +15,70 @@ The command creates a new `overrides.ts` file under `amplify/backend/storage/<re
 ## Customize Amplify-generated S3 resources
 
 Apply all the overrides in the `override(...)` function. For example to enable versioning on your S3 bucket:
+
 ```ts
 import { AmplifyS3ResourceTemplate } from '@aws-amplify/cli-extensibility-helper';
 
 export function override(resources: AmplifyS3ResourceTemplate) {
   resources.s3Bucket.versioningConfiguration = {
-    status: "Enabled"
-  }
+    status: 'Enabled'
+  };
```

Comment:
semi-colon doesn't seem in line with other examples

Reply:
semi colon is required when adding a iam policy document